skills/unity-ui/SKILL.md

```
@@ -1,384 +1,181 @@
+u scene hierarchy --depth 3
+u component list -t "Canvas"
+u component inspect -t "MyButton" -T "UnityEngine.UI.Button"
 ```
 
-コンパイルエラーがなければ:
+## 開発→テストフロー
 
-```bash
-u play
-u state          # isPlaying == true まで（最大10秒）
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)